src/applications/discharge-wizard/components/v2/resultsComponents/AdditionalInstructions.jsx

```diff
@@ -0,0 +1,266 @@
+import React from 'react';
```

Comment:
This is the only new change for this PR. Other changes will resolve when I merge Step 3